flink-staging/flink-gelly/src/test/java/org/apache/flink/graph/test/operations/GraphCreationWithMapperITCase.java

```diff
@@ -52,16 +72,13 @@ public void testWithDoubleValueMapper() throws Exception {
 		Graph<Long, Double, Long> graph = Graph.fromDataSet(TestGraphUtils.getLongLongEdgeData(env),
 				new AssignDoubleValueMapper(), env);
 
-        DataSet<Vertex<Long,Double>> data = graph.getVertices();
-        List<Vertex<Long,Double>> result= data.collect();
-		
+		graph.getVertices().writeAsCsv(resultPath);
```

Comment:
Same here.. We changed the tests to use `collect()` instead of files in #863. Please don't change it back ;)